tests/topotests/bgp_evpn_vxlan_topo1/PE2/evpn.vni.json

```diff
@@ -1,12 +1,16 @@
 {
   "vni":101,
   "type":"L2",
-  "vrf":"default",
+  "tenantVrf":"default",
   "vxlanInterface":"vxlan101",
   "vtepIp":"10.30.30.30",
   "mcastGroup":"0.0.0.0",
   "advertiseGatewayMacip":"No",
-  "numRemoteVteps":[
-    "10.10.10.10"
-  ]
+  "numRemoteVteps":1,
```

Comment:
This is also a breaking change?

Reply:
Yes, but in Tuedays' technical meeting it was decided to fix this as its totally wrong to have numRemoteVteps as list.
@donaldsharp  and others in call agree to not have backward for this but rather have for tenant vrf case.